### PR TITLE
feat(docs): showcase page

### PR DESCRIPTION
**Pull request by ZarcoNontol**

### CODE REVIEW

File: packages/docs/src/components/HomepageFeatures.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const FeatureList = [
   },
 ];
 
-function Feature({Svg, title, description}) {
```
Comment (wittjosiah) [inline packages/docs/src/components/HomepageFeatures.tsx:35]: suggestion: if you're going to convert to tsx then add the types, otherwise just leave it as js for the time being

File: packages/docs/src/components/ShowcaseCard.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  return (
+    <Card 
+      sx={{
+        flex: 1,
+        '& .MuiCardActionArea-root:hover': {
+          color: 'inherit',
+          textDecoration: 'none'
+        }
+      }}
+      variant="outlined">
```
Comment (wittjosiah) [inline packages/docs/src/components/ShowcaseCard.tsx:26]:
```suggestion
      variant="outlined"
      sx={{
        flex: 1,
        '& .MuiCardActionArea-root:hover': {
          color: 'inherit',
          textDecoration: 'none'
        }
      }}
    >
```

Comment (wittjosiah) [reply packages/docs/src/components/ShowcaseCard.tsx:26]: suggestion: Add in a placeholder image along the lines of Rich's comment on discord

**Review by wittjosiah — COMMENTED** [review]
I think we should make sure we pull in our linter to this repo soon to help manage it a bit more automatically. Probably in a separate PR as I suspect it will have a lot of notes about our code.

File: packages/docs/docusaurus.config.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ const darkCodeTheme = require('prism-react-renderer/themes/okaidia');
 const config = {
   title: 'DXOS',
   tagline: 'The Decentralized Operating System',
-  url: 'https://dxos.org',
+  url: 'https://qa.dxos.org/',
```
Comment (wittjosiah) [inline packages/docs/docusaurus.config.js:13]: can you make this dependent on an env var? We'll need this to change depending on which place it's being deployed

File: packages/docs/docusaurus.config.js [hunk 4]
```diff
@@ -55,7 +55,8 @@ const config = {
         logo: {
           alt: 'DXOS',
           src: 'img/dxos-horizontal.svg',
-          srcDark: 'img/dxos-horizontal-white.svg'
+          srcDark: 'img/dxos-horizontal-white.svg',
+          target: "_blank"
```
Comment (wittjosiah) [inline packages/docs/docusaurus.config.js:59]: suggestion: Add a comment here on why this is required so it doesn't get removed unexpectedly

File: packages/docs/package.json [hunk 5]
```diff
@@ -6,7 +6,8 @@
     "docusaurus": "docusaurus",
     "start": "docusaurus start --no-open",
     "dev": "yarn start",
-    "build": "docusaurus build",
+    "build:public": "rm -rf public && mkdir public && cp -r build public/",
```
Comment (wittjosiah) [inline packages/docs/package.json:9]: why is this needed on top of the docusaurus build?

Comment (ZarcoNontol) [reply packages/docs/package.json:9]: Sorry this was a test, I will remove it

File: packages/docs/docusaurus.config.js (hunk 4 above)
Comment (ZarcoNontol) [reply packages/docs/docusaurus.config.js:61]: Good catch!

**Review by wittjosiah — APPROVED** [review]
